app/controllers/people_controller.rb

```diff
@@ -78,9 +80,7 @@ def show
         gon.preloads[:photos] = {
           count: Photo.visible(current_user, @person).count(:all)
         }
```

Comment:
For the sake of unification: could you also change this like you did that for the `contacts_count`? This would also include a change in the profile header view.